app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_users_posts_counter
```

Comment:
- [ ] [OPTIONAL] Please use the callback `after_save` to call the `update_users_posts_counter`, so the method is called automatically every time a new post has been saved to the database, then make the `update_users_posts_counter` method private.